src/components/NotificationModal.tsx

```diff
@@ -230,3 +244,41 @@ const ExtraFieldContainer = styled.div`
 const ErrorMessage = styled(Body)`
   color: ${uiColors.red.base};
 `;
+
+const borderBottom = `1px solid ${uiColors.gray.light2};`;
+
+const Section = styled.div`
+  padding-bottom: 24px;
+  margin-bottom: 22px;
+  border-bottom: ${borderBottom};
```

Comment:
`borderBottom` has a semicolon at the end of the string so when the css is generated it will have 2 semicolons at the end of this style. You should remove one of the semi-colons.